criu/cr-restore.c

```diff
@@ -1781,10 +1781,13 @@ static int create_children_and_session(void)
 static int restore_task_with_children(void *_arg)
 {
 	struct cr_clone_arg *ca = _arg;
+	struct pstree_item *parent;
+	u32 current_cg_set = 0;
 	pid_t pid;
 	int ret;
 
 	current = ca->item;
+	parent = current->parent;
```

Comment:
possible remove this line and move the assignment to the declaration line (don't use current then)